tests/factories.py

```diff
@@ -2,135 +2,167 @@
 """Factory classes for easily generating test objects."""
 
 from __future__ import absolute_import
+from __future__ import unicode_literals
 
 import random
-import datetime
 
 import factory
+import faker
 
-from h.api import storage
+from h import db
+from h import models
 from h.accounts import models as accounts_models
```

Comment:
I would quite like to send a follow-up pr to sort out the `User` factory though. It's fine, but it could use faker to get nicer fake usernames and passwords etc